app/assets/javascripts/supplemental_files.js

```diff
@@ -90,6 +92,11 @@ $('.supplemental-file-form')
 
     // Show flash message for success
     $row.find('.message-content').html('Successfully updated.');
+    // Show/hide icon based on the updated machine-generated form check
+    isMachineGen
+      ? $row.find('.fa-laptop').removeClass('d-none')
+      : $row.find('.fa-laptop').addClass('d-none');
+    $row.find('.fa-laptop').removeClass('d-none');
```

Comment:
Why is there this second `removeClass` after the `removeClass`/`addClass`?

Reply:
I've missed that when cleaning things up. I'll remove it.